src/components/iOSDownloadButton.js

```diff
@@ -2,8 +2,13 @@ import React from "react"
 import iOSDownloadImage from "src/images/iOsBadgeBanner.svg"
 
 const IOSDownloadButton = props => (
-  <a href="https://example.com" target="_blank" {...props}>
-    <img src={iOSDownloadImage}></img>
+  <a
```

Comment:
Can you get me this Eslint settings?

Reply:
yes

https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-no-target-blank.md